src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main() {
+        Machine [] machines = new Machine[]{new Bulldozer(),new Truck(),new Excavator()};
```

Comment:
lets stick with the same code style 
```suggestion
        Machine[] machines = new Machine[] {new Bulldozer(), new Truck(), new Excavator()};
```